### ISSUE TITLE
Extensions view: enter does not move focus to extensions description

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#117595]
Found by @jvesouza

1. Keyboard focus onto the installed extensions list
2. Move up / down, press enter 
3. Focus is not moved to the extensions details 🐛 

This is working in current stable.
This is important for screen reader users so they can browse extensions description

fyi @mjbvz since I know he did some changes to web views

**Comment by sandy081** [comment]
Assigning to @mjbvz because I can confirm that focus call is being made to web view/

**Comment by mjbvz** [comment]
I don't see this working in stable either. Test by both trying to use the arrow keys to scroll the preview and looking at document.activeElement

**Comment by sandy081** [comment]
@mjbvz I think it works in Stable, I can open the extension editor and scroll the details page up/down all using keyboard and the same does not work in Insiders

**Stable**

![Kapture 2021-02-25 at 09 15 19](https://user-images.githubusercontent.com/10746682/109123574-4abb3480-774a-11eb-8eea-3e5fd0ad0e9c.gif)

**Insider**

![Kapture 2021-02-25 at 09 16 13](https://user-images.githubusercontent.com/10746682/109123611-5575c980-774a-11eb-8205-54a752a5c0ce.gif)

**Comment by isidorn** [comment]
@mjbvz it would be great if we could fix for this milestone. Assigning to February, and if you do not have time, you can push out to march and I will let the accessibility community know.

**Comment by mjbvz** [comment]
Ah, I was using iframe based webviews which apparently don't have this issue

Will take a look since this effects the markdown preview too

**Comment by isidorn** [comment]
Thanks for fixing! I will verify this once the build is out

**Comment by isidorn** [comment]
This works great now, adding verified label

**Comment by jvesouza** [comment]
This works great, thanks all.
